app/src/ui/history/commit-summary.tsx

```diff
@@ -357,42 +345,23 @@ export class CommitSummary extends React.Component<
             </li>
             {this.renderTags()}
 
-            {enableSideBySideDiffs() || (
-              <li
-                className="commit-summary-meta-item without-truncation"
-                title="Hide Whitespace"
-              >
-                <Checkbox
-                  label="Hide Whitespace"
-                  value={
-                    this.props.hideWhitespaceInDiff
-                      ? CheckboxValue.On
-                      : CheckboxValue.Off
-                  }
-                  onChange={this.onHideWhitespaceInDiffChanged}
-                />
-              </li>
-            )}
```

Comment:
This was here because before side-by-side diff, the "Hide Whitespace" option was a checkbox, that then was replaced by the ⚙️  Diff Options button/popup.